utils/svg.go

```diff
@@ -0,0 +1,71 @@
+package utils
+
+import "html/template"
+
+func IncludeSvg(name string) template.HTML {
```

Comment:
Not all svgs are part of new svg logic (like `svg.go`). For example:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/c4e270adfe9d6a04eef25a05e1c3412f96785806/templates/advertisewithus.html#L53
or
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/3683ab1845eaf14fcef346d7364b1aaaaacaeef7/templates/charts.html#L287

Should this be changed to harmonize everything further?

Reply:
Not necessary, its main purpose is if you have dynamic svg's coming from the backend. 
But yes if some SVG's are used more then once in would also make sense to put them in the svg.go. 
And of course we can also discuss if all inline SVG's should be handled that way - but that can be done in the future I guess.

Reply:
Personally I like that removing lenghty svg code from any html file improves the readability quite a bit. But sure, this is out of scope for the current ticket. Just keep it in the back of your head :)